projects/opendr_ws/src/perception/scripts/object_detection_2d_detr.py

```diff
@@ -24,14 +24,17 @@
 
```

Comment:
I made some changes on this file to resolve the conflicts, so it should be good to go.